lib/Rex/CLI.pm

```diff
@@ -517,8 +519,15 @@ CHECK_OVERWRITE: {
 
     my %groups = Rex::Group->get_groups;
     _print_color( "Server Groups\n", "yellow" ) if ( keys %groups );
+    my ($cols) = Term::ReadKey::GetTerminalSize(*STDOUT);
+    $Text::Wrap::columns = $cols // 80;
```

Comment:
@kablamo: more specifically we need to maintain perl-5.8.9 compatibility until CentOS/RHEL 5 EOL. You can run test for that locally before committing/pushing with `dzil test --no-author --release`.